src/cf-tables/src/cf-table-sortable.js

```diff
@@ -41,8 +40,9 @@ var TableSortable = {
     this.bindProperties();
     if ( this.ui.sortButton ) {
       this.sortColumnIndex = this.getColumnIndex();
-      this.sortDirection = this.contains( this.ui.sortButton, this.classes.sortDown )
-      					   ? DIRECTIONS.DOWN : DIRECTIONS.UP;
+      this.sortDirection =
+        this.contains( this.ui.sortButton, this.classes.sortDown ) ?
+        DIRECTIONS.DOWN : DIRECTIONS.UP;
```

Comment:
This is an awkward way to format a ternary statement but if that's what eslint did then ¯\_(ツ)_/¯.

Reply:
Yeah since the ternary operator has a colon on the line it's used it wants the ? on the line before so some of it is on both lines where it has an impact.

Reply:
Death to ternaries!